# RooverFeedback.msg
uint16 start
int16 cmd1
int16 cmd2
int16 speed_l 		# rpm links
int16 speed_r 		# rpm rechts
int16 wheel_l_cnt    	# teller links
int16 wheel_r_cnt    	# teller rechts
int16 battery_voltage
int16 board_temp
int16 cmd_led
uint16 checksum